src/Version/version.api.ts

```diff
@@ -0,0 +1,36 @@
+import globalAxios, { AxiosInstance, AxiosPromise, AxiosRequestConfig } from 'axios';
+
+import { RequestArgs } from '../api';
+import { Configuration } from '../configuration';
+import { getRequestOptionsWithApiKey } from '../lib/get-request-options-with-api-key';
+import { VersionAPIAxiosParamCreator, VersionAPI } from './version.api.types';
+
+const versionApiAxiosParamCreator = (configuration: Configuration): VersionAPIAxiosParamCreator => {
+  return {
+    getVersionRequestArgs: async (options: AxiosRequestConfig = {}): Promise<RequestArgs> => {
+      const requestPath = `/api/Version`;
+      const requestOptionsWithHeaders = await getRequestOptionsWithApiKey('GET', configuration, options);
+
+      return {
+        url: requestPath,
+        options: requestOptionsWithHeaders,
+      };
+    },
+  };
+};
+
+export const VersionApi = (configuration: Configuration): VersionAPI => {
```

Comment:
This should be exported at the end of the file. The naming of these files and the `Version` directory doesn't seem to match the rest of the project.

Reply:
I took a swing at simplifying this (renamed the folder to `version-api`), and lowercased the function definition here. I had considered making this a class, and I think that I just kept thinking about it that way and kept the uppercase here which definitely makes this feel weird.